src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/Delegation/TestDelegatedCompletionListProvider.cs

```diff
@@ -37,41 +36,37 @@ private TestDelegatedCompletionListProvider(
     }
 
     public static TestDelegatedCompletionListProvider Create(
-        ILoggerFactory loggerFactory,
-        params DelegatedCompletionResponseRewriter[] responseRewriters) =>
-        Create(delegatedCompletionList: null, loggerFactory, responseRewriters: responseRewriters);
+        ILoggerFactory loggerFactory) =>
+        Create(delegatedCompletionList: null, loggerFactory);
```

Comment:
nit: please move `=>` to start of next line. I think that's pretty consistent with other expression-bodied members throughout the repo.